File: Golang/2219-maximum-number-of-words-found-in-sentences/maximum-number-of-words-found-in-sentences.go
func mostWordsFound(sentences []string) int {
    maxCount := 0
    for i := 0; i < len(sentences); i++ {
        count := 1
        for j := 0; j < len(sentences[i]); j++ {
            if sentences[i][j] == ' ' {
                count += 1
            }
        }
        if count > maxCount {
            maxCount = count
        }
    }
    return maxCount
}

File: Python/2219-maximum-number-of-words-found-in-sentences/maximum-number-of-words-found-in-sentences.py
class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        max_words = count_words = 0
        for sentence in (sentences):
            count_words = 1
            for char in sentence:
                if char == ' ':
                    count_words += 1
            if count_words > max_words:
                max_words = count_words
        return max_words